internal/compiler/passes/lower_popups.rs

```diff
@@ -125,6 +138,52 @@ fn lower_popup_window(
     let coord_x = NamedReference::new(&popup_comp.root_element, "x");
     let coord_y = NamedReference::new(&popup_comp.root_element, "y");
 
+    // Take a reference to the close policy
+    let close_policy = close_policy
+        .map(|b| {
+            let b = b.into_inner();
+            (b.expression, b.span)
+        })
+        .or_else(|| {
+            let mut base = popup_window_element.borrow().base_type.clone();
+            while let ElementType::Component(b) = base {
+                base = b.root_element.borrow().base_type.clone();
+                if let Some(binding) = b.root_element.borrow().bindings.get(CLOSE_POLICY) {
+                    let b = binding.borrow();
+                    return Some((b.expression.clone(), b.span.clone()));
```

Comment:
If this is the fallback handling, then shouldn't this be done before like 85?

The same applies to the close_on_click: The check if both are set should IMO be done when we've concluded if a binding exists, either in the popup_window_element or the root if it's inherited.